opts/global.go

```diff
@@ -547,6 +547,11 @@ type AxisLabel struct {
 	LineHeight string `json:"lineHeight,omitempty"`
 }
 
+type AxisTick struct {
+	Show     bool   `json:"show"`
```

Comment:
Any comments?

Reply:
Comments are added. I put an axis line option that's available for y-axis as wel